bundle/bundle.go

```diff
@@ -17,20 +17,32 @@ import (
 var (
 	//go:embed *
 	regalBundle    embed.FS
-	devPath        = os.Getenv("REGAL_BUNDLE_PATH")
 	lastErrMsg     = atomic.Pointer[string]{}
 	EmbeddedBundle = sync.OnceValue(func() *bundle.Bundle {
 		return rio.MustLoadRegalBundleFS(regalBundle)
 	})
 	successLogOnce = sync.OnceFunc(func() {
 		fmt.Fprintln(os.Stderr, "Successfully loaded development bundle")
 	})
+	Dev = &DevSettings{}
 )
 
-func init() {
-	if devPath != "" {
-		fmt.Fprintln(os.Stderr, "REGAL_BUNDLE_PATH set. Will attempt using development bundle from:", devPath)
-	}
+type DevSettings struct {
+	bundlePath string
+	mux        sync.RWMutex
+}
+
+func (ds *DevSettings) SetBundlePath(path string) {
```

Comment:
Unused?

Reply:
Almost, but not quite :) 

<img width="1091" height="221" alt="Screenshot 2025-08-11 at 19 10 27" src="https://github.com/user-attachments/assets/561557c4-aec2-45d4-9bc8-ea39faaa8770" />